account-decoder/src/parse_token_extension.rs

```diff
@@ -0,0 +1,161 @@
+use {
+    crate::parse_token::UiAccountState,
+    spl_token_2022::{
+        extension::{self, BaseState, ExtensionType, StateWithExtensions},
+        solana_program::pubkey::Pubkey,
+    },
+};
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase", tag = "extension", content = "state")]
+pub enum UiExtension {
+    Uninitialized,
+    TransferFeeConfig(UiTransferFeeConfig),
+    TransferFeeAmount(UiTransferFeeAmount),
+    MintCloseAuthority(UiMintCloseAuthority),
+    ConfidentialTransferMint,
+    ConfidentialTransferAccount,
```

Comment:
Is it worth adding a note saying that these will be done later? Or leaving these as placeholders and then parse them to `Unparseable` for now?

Reply:
Both are great ideas! Will fixup

Reply:
Actually, I kind of prefer leaving these instead of returning `UnparseableExtension`. Once these parsers are implemented, the change will be additive -- adding the `state` field to the map in the response. Wdyt?
I will add comments regardless.